edit.js

```diff
@@ -0,0 +1,126 @@
+var dipendenti=[
+    { 
+        nome: "mario rossi",
+        competenzeAssociate: [
+            {
+                nome: 'java',
+                area: 'linguaggi programmazione',
+                livello: 'base'
+            },
+            {
+                nome: 'Linux',
+                area: 'Sistemi operativi',
+                livello: 'intermedio'
+            }
+        ]
+        },
+        { 
+        nome: "franco verdi",
+        competenzeAssociate: [
+            {
+                nome: 'js',
+                area: 'linguaggi programmazione',
+                livello: 'avanzato'
+            },
+            {
+                nome: 'Windows',
+                area: 'Sistemi operativi',
+                livello: 'base'
+            }
+        ]
+    }
+]
+var tempLivelli=['base', 'intermedio', 'avanzato']
+
+for(let i=0;i<dipendenti.length;i++){
+    document.getElementById("employee").innerHTML+='<option value='+i+'>'+dipendenti[i].nome+'</option>'
+}
+
+function showArea(){
+    document.getElementById("area").innerHTML='<option value="" selected="selected" disabled="disabled">Seleziona Area di Competenze</option>'
+    var employee=document.getElementById("employee").value
+    for(let i=0;i<dipendenti[employee].competenzeAssociate.length;i++){
```

Comment:
illeggibile... tirati fuori la variabile competenze associate

```suggestion
    const competenze = dipendenti[employee].competenzeAssociate;
    for(let i=0; i < competenze.length; i++) {
```

Reply:
e ricordati di mettere gli spazi di formattazione, per noi esseri umani si legge molto meglio, per le macchine esistono i minificatori